ledger-core/virtual/integration/delegationtoken_test.go

```diff
@@ -334,7 +335,7 @@ func TestDelegationToken_IsMessageFromVirtualLegitimate(t *testing.T) {
 				// increment pulse for VStateReport and VDelegatedCallRequest
 				server.IncrementPulse(ctx)
 
-				switch testCase.approverVE {
+				switch testCase.expectedVE {
```

Comment:
because  
```
func (s service) getExpectedVE(ctx context.Context, subjectRef reference.Global, verifyForPulse pulse.Number) (reference.Global, error) {
	expectedVE, err := s.affinity.QueryRole(ctx, node.DynamicRoleVirtualExecutor, subjectRef.GetLocal(), verifyForPulse)
```